Automattic/wp-calypso
# Editor Welcome: Add instructional graphics for mobile UI

### Changes proposed in this Pull Request

* Add mobile assets to Welcome Tour
 
#### Technical Changes
* Make use of `<picture/>` element to support different graphics on different viewports.
* Update `imgSrc` structure from string to map in order to support different graphics on different viewports.
* Update styling to maintain proportions of graphics

### Testing
#### Preparations 
1. Make sure that you're sandboxed
2. Apply D67026-code w/ `arc patch-image` (see PCYsg-h9w-p2)
3. Make sure that`s0.wp.com` is sandboxed. 
4. Check out this branch and `yarn && yarn dev --sync`. 

#### Testing instructions
1. Navigate to one of your sandboxed sites and clear the browser's local storage.
2. Navigate to the editor
3. The welcome tour should pop up. If not, launch it from the _more settings_ menu.
4. Validate that:
    - [ ] All animations work as expected (see screencap below).
    - [ ] The animations for desktop are visible.
5. Switch to a mobile viewport
6. Repeat steps 1-3 and validate that:
    - [ ] All animations work as expected (see screencap below).
    - [ ] The animations for mobile are visible. 

### Welcome Tour on desktop screencap
https://user-images.githubusercontent.com/12104589/133770360-8adf4aad-bac1-495c-8474-a9b39431d2a0.mov

### Welcome Tour on mobile screencap
https://user-images.githubusercontent.com/12104589/133770349-98cbdbfc-6456-49f4-8785-6cc3cc06e593.mov

Fixes #55826
